app/src/main/java/com/imeanttobe/consensusapp/ui/splash/SplashScreen.kt

```
@@ -46,6 +48,7 @@ fun SplashScreen(
                     inclusive = true
                 }
             }
+        } else if (splashState.value is UiState.Failure) {
         } else if (splashState.value is UiState.Failure) {
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyIfBlock)